packages/PriceLockup/PriceLockup.md

```diff
@@ -11,6 +11,8 @@
 - Use `rateText` prop to provide month or year
 - Use `topText` and `bottomText` props to provide additional information
 - Align the currency sign to the right when displaying prices in French
+- Use `strikethrough` option for price savings comparison.
+  - If using `strikethrough` pricing, it must be accompanied by corresponding `ariaLabel` to be used by screen readers
```

Comment:
Can we add a warning if `strikethrough` is used without `ariaLabel`. See https://github.com/telus/tds-core/blob/master/packages/PriceLockup/PriceLockup.jsx#L147 for an example.